web-app-aspnet/appsettings.json

```diff
@@ -1,20 +1,22 @@
 {
-  "AzureAd": {
-    "Instance": "https://login.microsoftonline.com/",
-    "TenantId": "Enter the tenant ID obtained from the Azure portal",
-    "ClientId": "Enter the client ID obtained from the Azure portal",
-    "ClientCertificates": [
-      {
-        "SourceType": "StoreWithThumbprint",
-        "CertificateStorePath": "CurrentUser/My",
-        "CertificateThumbprint": "Enter the certificate thumbprint obtained from the Azure portal"
-      }   
-    ],
-    "CallbackPath": "/signin-oidc"
-  },
+"AzureAd": {
+  "Instance": "https://login.microsoftonline.com/",
+  "TenantId": "Enter the tenant ID obtained from the Microsoft Entra Admin Centre",

```

Comment:
This needs to be consistent in both capitalization and use of the word "center" (not centre, unless we're talking about a localized version). Notice that previously in the documentation you refer to it as "Microsoft Entra admin center."